### ISSUE TITLE
New to this

### ISSUE THREAD

**Issue by iCylow** [atomjack/dubtrackapi#11]
So I downloaded the dubtrackapi master file and then placed a file titled bot.js inside, pasted in the code provided and changed out the username/password and room location. Did the phantom install (npm install phantomjs) and now when I go to do 'node bot.js' I get the error that it "cannot find the module './dubtrackapi'". I'm kind of stuck here please help :/

**Comment by iCylow** [comment]
I realize I'm probably missing something quite simple but I'm a bit lost

**Comment by ImGrumpy** [comment]
For me, I had to remove the "./" that's shown in the example.

`var DubtrackAPI = require('dubtrackapi');`

**Comment by iCylow** [comment]
well now it gives me the error that if cant find the dubtrackapi module

**Comment by iCylow** [comment]
Yeah I thought that might do it but I dont think so, still probably missing something simple

**Comment by Fuechschen** [comment]
I would advise you to use a slightly other way. Create a new folder and place your bot,js in it. create another file named package.json that looks like this: https://share.fuechschen.org/d/131020150245.package.json

now run 'npm install' in that folder. you now should be able to start your bot with 'node bot.js'.

**Comment by atomjack** [comment]
You don't put your bot.js in the same directory as dubtrackapi - make a new folder, and put bot.js in it, and also put the dubtrackapi directory in there as well - so that both bot.js and dubtrackapi are in the same directory. You might then also cd into the dubtrackapi directory and run `npm install` to install all the dependencies.
